ibm/mas_devops/roles/suite_db2_setup_for_manage/tasks/main.yml

```diff
@@ -74,6 +74,29 @@
   delay: 30 # seconds
   until: copy_result.rc == 0
 
+- name: Execute db2 restart database
+  shell: |
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2stop"
+    sleep 180
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2start"
+    sleep 180
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2 restart database {{ db2_dbname }}"
+    sleep 180
+    oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su - db2inst1 -lc "db2 activate database {{ db2_dbname }}"
+    sleep 180
```

Comment:
Is there not another way to see the if the databse is not activated rather than just hoping that 180 seconds was long enough? Same question for the other commands

Reply:
@whitfiea Please note that with this code, we have tested 15+ deployment of Manage and we were able to deploy it successfully.

Reply:
@whitfiea Tested with updating this 10 sec + updating the pause to 1 minutes (from 3 minutes)
/tmp/setupdb.sh executed fine. Complete logs https://github.ibm.com/maximoappsuite/tracker-hyperscalers/issues/271#issuecomment-55343837
![image](https://user-images.githubusercontent.com/45054700/229186986-d66424f9-c0ac-4992-8065-4456c3492049.png)